src/compiler/rust/canister_methods/src/upload_file_chunk/mod.rs

```diff
@@ -15,18 +15,14 @@ pub fn get_upload_file_chunk() -> proc_macro2::TokenStream {
         #check_if_latest_version_src
         #hash_file_src
 
-        #[ic_cdk_macros::update]
+        #[ic_cdk_macros::update(guard = is_authenticated)]
```

Comment:
Do you have tests for each of the functions with this guard to ensure that unauthorized users can't access it?